src/pages/projects.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import Hero from '../components/Hero'
+import Heading from '../components/Heading'
+import CardBlock from '../components/CardBlock'
+import Card from '../components/Card'
+import Layout from '../components/Layout'
+
+const StyledSection = styled.section`
+  background-color: var(--light-grey);
+`
+
+const ProjectsPage = ({ data }) => {
+  const activeRaw = []
+  const inactiveRaw = []
+  // console.log(data.allMarkdownRemark.nodes[0].frontmatter.thumbnail.childImageSharp.fluid.src)
+  //loop through data and separate out active and inactive projects
+
+  for (const project of data.allMarkdownRemark.nodes) {
+    if (project.frontmatter.areas.toLowerCase() === 'not actively recruiting') {
+      inactiveRaw.push(project)
+    } else {
+      activeRaw.push(project)
+    }
+  }
+
+  //loop through activeRaw and create new objects to match props format of CARD component
+
+  let newActiveList = []
+  for (const active of activeRaw) {
```

Comment:
I was thinking that because i needed to create 2 new arrays that just interating and separating out the elements would  use only 1 loop through.  Unless I'm just not being imaginative enough with how a map would work here.